web/src/app/services/ServiceDeleteDialog.js

```diff
@@ -61,34 +63,42 @@ export default function ServiceDeleteDialog({ serviceID, onClose }) {
     variables: { id: serviceID },
   })
   const input = [{ type: 'service', id: serviceID }]
-  const dispatch = useDispatch()
   const [deleteService, { loading, error }] = useMutation(mutation, {
```

Comment:
I'd much prefer calling out `deleteServiceStatus` here instead of scraping the variables we need, as it results in easier to read code later in the component, i.e. situations like on L95 where we are doing `loading || dataLoading`. IMO `loading` should always refer to loading data in, and we can use `status` to identify that we are looking at mutation updates

Reply:
The dialogs primary action is the mutation, and some need data and some don't, which is why I went with `loading` and `dataLoading` without thinking about it, but clear is always better.

I disagree with just using `status` for the query though as it seems just as ambiguous (status of what?). Maybe when there is more than one query and/or mutation, we should always make it clear?

What do you think of:
```js
const [data, dataStatus] = //...
const [deleteService, deleteServiceStatus] = //...
```

Just follow the first arg name for the second, unless we have a single query, and no mutation, then the shorthand `[data, {loading, error}]` would be appropriate. 

What do you think?

Reply:
I mean keeping loading/error/data reserved for the main `query` if ever needed, and the `status` _suffix_ (didn't mean as a whole variable, mb) for mutations like `updateRotationStatus`, etc. But I see what you mean here, and that works. When mutations are present I like being explicit with `xxStatus.loading`, though

Reply:
Awesome, will make the change